### PR TITLE
Wiki AtragMx updated_3

### PR DESCRIPTION
**Pull request by Laid3acK**
- Add an example with `Add New Gun` in `GunList`
- link with ACE3 latitude
- Minor updates
- Links with AB tools
- Direct links with official manual

### CODE REVIEW

**Comment by Laid3acK** [comment]
Certainly my last commit because my first intention was not a bla bla wiki, 
except if you want some modifications.

File: docs/wiki/feature/atragmx.md [hunk 1]
```diff
@@ -55,19 +61,23 @@ Horus ATragMX software considers atmospheric conditions, gun data, ammunition, r
 - Optionally, `Save Gun` and `Done` in the `GunList`.
  
 **In position:**
+
 - Update the `Atmsphr` column with the [Kestrel 4500]({{ site.baseurl }}/wiki/feature/kestrel4500.html) and `Done`.
 - *Check the new `Muzzle Velocity` in the `Gun` column.*
 - Update the `Target` column (the [wind arrow]({{ site.baseurl }}/wiki/feature/weather.html) will also help). [[1]](https://horusvision.com/download/manual_Horus_ATrag-v385.pdf#page=16) [[2]](https://horusvision.com/download/manual_Horus_ATrag-v385.pdf#page=30) [[3]](https://horusvision.com/download/manual_Horus_ATrag-v385.pdf#page=32) [[4]](https://horusvision.com/download/manual_Horus_ATrag-v385.pdf#page=33)
+- *The latitude for all common maps can be found in the [ACE3 Github](https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_getMapData.sqf).*
```
Comment (bux) [inline docs/wiki/feature/atragmx.md]: use the `{{ xxx.github... url }}` thing ... :)

Comment (Laid3acK) [inline docs/wiki/feature/atragmx.md]: Like that, correct ?
`{{ site.ace.githubUrl }}/blob/master/addons/common/functions/fnc_getMapData.sqf`

**Comment by Laid3acK** [comment]
- Fix the link (?)
- Add a quick note for advanced users (i forgot it)

**Comment by Laid3acK** [comment]
There is a problem with this PR, i made some mistakes ?

**Comment by bux** [comment]
thanks for reminding me @Laid3acK

**Comment by Laid3acK** [comment]
Many thanks to the @ace3mod team for your precious help.

**Pull request merged**